scripts/write_to_db.py

```diff
@@ -0,0 +1,47 @@
+import pandas as pd
+import numpy as np
+from spreadsheets_helper import read_spreadsheet
+from database_helper import connect_to_db, update_row_orm, close_connection
+
+def main():
+    SCOPES = ['https://www.googleapis.com/auth/spreadsheets']
+    SPREADSHEET_ID_INPUT = '1RZDZTswZxI-smwnyi8Oh5ck092hgl_CBxjtQURrczBs'
+    SPREADSHEET_RANGE ='A1:Z10'
+    credentials_file_name = 'credentials.json'
+    DATABASE_URL = ''
+
+    df = read_spreadsheet(SPREADSHEET_ID_INPUT, SPREADSHEET_RANGE, SCOPES)
+    df['id'] = df['id'].astype(int)
+    db_session = connect_to_db(DATABASE_URL)
+
+    for index, row in df.iterrows():
+        store_name = row['name']
+        flagged = row['flag']
+        deleted = row['deleted']
+        reviewed = row['Reviewed']
+        useful_info = row['availabilityInfo'].strip().lower() if row['availabilityInfo'] else ''
+        safety_info = row['safetyInfo'].strip().lower() if row['safetyInfo'] else ''
+
+        update_dict = {
+            'flag': row['flag']
+        }
+
+        # If deleted is marked true, then it has been reviewed
+        if deleted == 'TRUE':
+            update_dict['deleted'] = True
+            update_dict['reviewed'] = True
+
+        if row['openingTime']:
```

Comment:
why are we checking for only these keys and not the others? also, we seem to be doing this check in multiple places (like inside `update_row_orm`), which is not a good idea - since these keys might change over time, and we would need to remember to do them at multiple points. can we simplify this?

Reply:
if they are to be repeated, i think we should create methods on the `Store` and `Update` classes that encapsulate this implementation and expose it as a method to consumers

Reply:
Yep yep, all good points. There's a bunch of improvements that can be made. I wanted to get a script out that works. We can keep refactoring and improving it. I wanted to restrict which fields we change, that's why this check. Agreed that doing them through specific methods is cleaner!